### ISSUE TITLE
Different mixup loss function between code and paper

### ISSUE THREAD

**Issue by lukk47** [facebookresearch/mixup-cifar10#18]
The mixup loss function in code is as below:
![image](https://user-images.githubusercontent.com/24246633/56557971-a6a55700-65cf-11e9-8eb3-c537bf74a591.png),
while the mixture should be down before feed into the loss function according to the paper.

Will these two loss functions have the same results?

**Comment by songzhaozhe** [comment]
Same question. I do not think they are equivalent.

**Comment by kleinzcy** [comment]
@LokLu It is the same.

![image](https://user-images.githubusercontent.com/35723500/70629566-2702f000-1c65-11ea-853c-cde80c1e590f.png)

So the two loss functions are the same.

Let me know if I am wrong.

**Comment by lukk47** [comment]
@kleinzcy Your equations are correct. But the problem is that the 'pred' in your code are the logits output of the model instead of the softmax of pred.

**Comment by lizc126** [comment]
> @kleinzcy Your equations are correct. But the problem is that the 'pred' in your code are the logits output of the model instead of the softmax of pred.

Hi, just saw this and I am curious as well. But the criterion should ideally take the logits output instead of softmax of pred right?
